### PR TITLE
Add Solution e2e tests added

### PR DESCRIPTION
**Pull request by EnitanOO**
AB#14692
AB#15461

### CODE REVIEW

File: tests/NHSD.GPIT.BuyingCatalogue.E2E.PublicBrowseTests/Objects/Admin/AddSolutionObjects.cs [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+
+namespace NHSD.GPIT.BuyingCatalogue.E2ETests.Objects.Admin
+{
+    internal static class AddSolutionObjects
+    {
+        internal static By SolutionName => By.Id("SolutionName");
+        internal static By SupplierName => By.Id("SupplierId");
+        internal static By SolutionFrameworks => CustomBy.DataTestId("framework-names");
+        internal static By SaveSolutionButton => By.Id("Submit");
+        internal static By FoundationSolution => CustomBy.DataTestId("foundation-solution");
```
Comment (PhysicsJon) [inline tests/NHSD.GPIT.BuyingCatalogue.E2E.PublicBrowseTests/Objects/Admin/AddSolutionObjects.cs:12]: Each of these properties should have an extra line between them

File: tests/NHSD.GPIT.BuyingCatalogue.E2E.PublicBrowseTests/Actions/Admin/AddSolution.cs [hunk 2]
```diff
@@ -0,0 +1,71 @@
...
+                Wait.Until(d => d.FindElements(Objects.Admin.AddSolutionObjects.SolutionFrameworks).Count == 2);
+                return true;
+            }
+            catch
+            {
+                return false;
+            }
+        }
+
+        public bool GetFoundationSolution()
```
Comment (PhysicsJon) [inline tests/NHSD.GPIT.BuyingCatalogue.E2E.PublicBrowseTests/Actions/Admin/AddSolution.cs:38]: You aren't getting the value of the Foundation Solution checkbox here, so the name should be changed to reflect that you're only checking that it's displayed

**Review by PhysicsJon — CHANGES_REQUESTED** [review]

File: tests/NHSD.GPIT.BuyingCatalogue.E2E.PublicBrowseTests/Objects/Admin/AddSolutionObjects.cs (hunk 1 above)
Comment (EnitanOO) [reply tests/NHSD.GPIT.BuyingCatalogue.E2E.PublicBrowseTests/Objects/Admin/AddSolutionObjects.cs:12]: Done

File: tests/NHSD.GPIT.BuyingCatalogue.E2E.PublicBrowseTests/Actions/Admin/AddSolution.cs (hunk 2 above)
Comment (EnitanOO) [reply tests/NHSD.GPIT.BuyingCatalogue.E2E.PublicBrowseTests/Actions/Admin/AddSolution.cs:38]: Done

**Review by PhysicsJon — APPROVED** [review]

**Pull request merged**